src/content/recipes/react-i18next.md

```diff
@@ -54,7 +42,7 @@ To expose this instance to Storybook, we can import it into the `./.storybook/pr
 import i18n from '../src/i18n';
 ```
 
-### 2. Wrap your stories with the i18next provider
+## 2. Wrap your stories with the i18next provider
 
 Now that Storybook has access to i18next, we need to share that with our stories. To do that we’re going to make a decorator to wrap our stories in.
 
```

Comment:
The proceeding code snippet needs to use `jsx`. [See related comment about Callout considerations.](https://github.com/storybookjs/frontpage/pull/616/files#r1383767253)